<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sanescan::MainWindow</class>
 <widget class="QMainWindow" name="sanescan::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaneScan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSplitter" name="splitter_vertical">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QTabWidget" name="tabs">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_scan">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <attribute name="title">
                <string>Scanning</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QStackedWidget" name="stack_settings">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="page_loading">
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_loading">
                      <property name="text">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Loading scanners&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Please wait...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="v_spacer1">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_settings">
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                     <widget class="sanescan::ScanSettingsWidget" name="settings_widget" native="true"/>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_ocr">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <attribute name="title">
                <string>OCR</string>
               </attribute>
               <layout class="QVBoxLayout" name="vertical_layout_8">
                <item>
                 <widget class="sanescan::OcrSettingsWidget" name="ocr_settings" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_ocr_progress">
                  <property name="text">
                   <string>OCR is in progress ...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_blurry_warning">
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>160</red>
                        <green>161</green>
                        <blue>162</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Page is blurry, OCR quality will be reduced</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
             <widget class="sanescan::ImageWidget" name="image_area" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="sanescan::PageListView" name="page_list" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_save">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="action_save_current_image"/>
    <addaction name="action_save_all_pages"/>
    <addaction name="action_save_all_pages_with_ocr"/>
   </widget>
   <addaction name="menu_save"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_save_current_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Current image</string>
   </property>
  </action>
  <action name="action_save_all_pages">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>All pages (raw scans)</string>
   </property>
  </action>
  <action name="action_save_all_pages_with_ocr">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>All pages (with OCR results)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sanescan::ImageWidget</class>
   <extends>QWidget</extends>
   <header>image_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sanescan::ScanSettingsWidget</class>
   <extends>QWidget</extends>
   <header>scan_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sanescan::PageListView</class>
   <extends>QWidget</extends>
   <header>pagelist/page_list_view.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sanescan::OcrSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ocr_settings_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
